ramalama/sandbox.py

```
@@ -86,7 +95,8 @@ def is_tty_cmd(self) -> bool:
         return getattr(self.args, "subcommand", "") == "sandbox"
 
     def add_network(self) -> None:
-        self.add_args(f"--network=container:{self.args.name}")  # type: ignore[attr-defined]
+        if getattr(self.args, "llm_endpoint") is None:
```

Ruff (0.15.18)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)